### PR TITLE
Min/Max Formula for Metrics Queries Update 

### PR DESCRIPTION
**Pull request by ouracheal**
### What does this PR do? 
Updates new min/max formula documentation in the dashboard querying page with an example of how the formula would be used in metrics query. Also linked the metrics explorer and SLOs pages to reference the dashboards querying page for further detail on formulas in metrics query. 

What is the motivation?
Reflecting launch of minimum/maximum formula in metrics query to GA.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/dashboards/querying/_index.md [hunk 1]
```diff
@@ -174,6 +174,16 @@ status:error / status:info
 
 **Note**: Formulas are not lettered. Arithmetic cannot be done between formulas.
 
+Here is an example using the `max` operator to find the maximum value between `status:error` and `status:ok`.
+
+```text
+max(status:error, status:ok)
+```
+
+{{< img src="dashboards/querying/arithmetic_7.png" alt="Formula example - max apm" style="width:75%;" >}}
```
Comment (estherk15) [inline content/en/dashboards/querying/_index.md:183]:
```suggestion
{{< img src="dashboards/querying/arithmetic_7.png" alt="Formula example for 'max' showing max count value between two APM queries" style="width:75%;" >}}
```

File: content/en/dashboards/querying/_index.md [hunk 2]
```diff
@@ -174,6 +174,16 @@ status:error / status:info
 
 **Note**: Formulas are not lettered. Arithmetic cannot be done between formulas.
 
+Here is an example using the `max` operator to find the maximum value between `status:error` and `status:ok`.
```
Comment (estherk15) [inline content/en/dashboards/querying/_index.md:177]: Fact check me here
```suggestion
Here is an example using the `max` operator to find the maximum count between `status:error` and `status:ok`.
```

File: content/en/metrics/explorer.md [hunk 3]
```diff
@@ -44,9 +45,9 @@ The possible options are:
 
 **Note**: The options may differ based on the metric type selected.
 
-### Functions
+### Functions/Formulas
 
-You can optionally add functions to your query using the function button. Not all functions are available for all metric types.
+You can optionally add functions to your query using the function button. Not all functions are available for all metric types. See the [querying][6] documentation to learn more about functions and formulas in queries.
```
Comment (estherk15) [inline content/en/metrics/explorer.md:50]:
```suggestion
You can optionally add functions to your query using the function button. Not all functions are available for all metric types. For more information, see the [querying][6] documentation. 
```

File: content/en/service_management/service_level_objectives/metric.md [hunk 4]
```diff
@@ -26,11 +26,11 @@ On the [SLO status page][5], select **New SLO +**. Then select [**Metric**][6].
...
-2. Use the `FROM` field to include or exclude specific groups using tags.
-3. For percentile-enabled DISTRIBUTION metrics, you must use the `count values...` aggregator to specify a numerical threshold for the metric to count. This feature is called Threshold Queries and allows you to count the number of raw values that match a numerical threshold to produce counts for your numerator and denominator. For more information, see [Threshold Queries][7].
-4. Optionally, for percentile-enabled DISTRIBUTION metrics, use the dropdown immediately to the right of the `count values..` aggregator to break your SLI out by specific groups.
-5. Optionally, for COUNT or RATE metrics, use the `sum by` aggregator to break your SLI out by specific groups.
+1. There are two queries to define. The numerator query defines the sum of the good events, while the denominator query defines the sum of the total events. Your queries must use COUNT, RATE, or percentile-enabled DISTRIBUTION metrics to ensure the SLO calculation behaves correctly. For more information, see [Querying][9] documentation. 
+3. Use the `FROM` field to include or exclude specific groups using tags.
```
Comment (estherk15) [inline content/en/service_management/service_level_objectives/metric.md:30]:
```suggestion
1. Use the `FROM` field to include or exclude specific groups using tags.
```

File: content/en/service_management/service_level_objectives/metric.md [hunk 5]
```diff
@@ -26,11 +26,11 @@ On the [SLO status page][5], select **New SLO +**. Then select [**Metric**][6].
...
+4. For percentile-enabled DISTRIBUTION metrics, you must use the `count values...` aggregator to specify a numerical threshold for the metric to count. This feature is called Threshold Queries and allows you to count the number of raw values that match a numerical threshold to produce counts for your numerator and denominator. For more information, see [Threshold Queries][7].
```
Comment (estherk15) [inline content/en/service_management/service_level_objectives/metric.md:31]:
```suggestion
1. For percentile-enabled DISTRIBUTION metrics, you must use the `count values...` aggregator to specify a numerical threshold for the metric to count. This feature is called Threshold Queries and allows you to count the number of raw values that match a numerical threshold to produce counts for your numerator and denominator. For more information, see [Threshold Queries][7].
```

File: content/en/service_management/service_level_objectives/metric.md [hunk 6]
```diff
@@ -26,11 +26,11 @@ On the [SLO status page][5], select **New SLO +**. Then select [**Metric**][6].
...
+5. Optionally, for percentile-enabled DISTRIBUTION metrics, use the dropdown immediately to the right of the `count values..` aggregator to break your SLI out by specific groups.
```
Comment (estherk15) [inline content/en/service_management/service_level_objectives/metric.md:32]:
```suggestion
1. Optionally, for percentile-enabled DISTRIBUTION metrics, use the dropdown immediately to the right of the `count values..` aggregator to break your SLI out by specific groups.
```

**Review by estherk15 — COMMENTED** [review]
Thanks @ouracheal, left some feedback, let me know if you have any questions!

File: content/en/dashboards/querying/_index.md (hunk 2 above)
Comment (ijkaylin) [inline content/en/dashboards/querying/_index.md:177]:
```suggestion
#### Minimum or Maximum between two queries

Here is an example using the `max` operator to find the maximum value between `status:error` and `status:ok`.
```

File: content/en/dashboards/querying/_index.md [hunk 7]
```diff
@@ -174,6 +174,16 @@ status:error / status:info
...
+
+```text
+max(status:error, status:ok)
```
Comment (ijkaylin) [inline content/en/dashboards/querying/_index.md:180]: Racheal, let's have a query that is either both metrics or metrics + another data source here

File: content/en/dashboards/querying/_index.md (hunk 1 above)
Comment (ijkaylin) [inline content/en/dashboards/querying/_index.md:183]: same comment - both metric queries or metrics / another data source. 

Make sure the example/screenshot you choose is an actual query that users would want to graph

File: content/en/metrics/explorer.md [hunk 8]
```diff
@@ -11,6 +11,7 @@ further_reading:
   - link: "/metrics/distributions/"
```
Comment (ijkaylin) [inline content/en/metrics/explorer.md:11]: @estherk15  distributions shouldn't be linked as a subpage under metrics explorer . It should be moved back as its own page like summary or metrics without limits is. I'd prefer if it existed right under metric types in terms of ordering

**Review by ijkaylin — CHANGES_REQUESTED** [review]

File: content/en/metrics/explorer.md (hunk 8 above)
Comment (estherk15) [reply content/en/metrics/explorer.md:11]: @ijkaylin I can do that! To clarify you want it at the same level under Metric Type? Or under as a child of Metric Type?

Comment (estherk15) [reply content/en/metrics/explorer.md:11]: https://datadoghq.atlassian.net/jira/software/projects/DOCS/boards/199

File: content/en/dashboards/querying/_index.md (hunk 2 above)
Comment (ouracheal) [reply content/en/dashboards/querying/_index.md]: added header

File: content/en/metrics/explorer.md (hunk 8 above)
Comment (ouracheal) [reply content/en/metrics/explorer.md:11]: Thanks for bringing that up @ijkaylin! 
I made a [request](https://datadoghq.atlassian.net/browse/DOCS-6447) for these changes on the board @estherk15 thank you for your help!

File: content/en/dashboards/querying/_index.md (hunk 7 above)
Comment (ouracheal) [reply content/en/dashboards/querying/_index.md:181]: Thanks for pointing that out @ijkaylin, this is the example I am planning to use and seen in our customer FR board. I am querying for the avg CPU usage within 10-second intervals for two zones and then identifying the maximum value between the two zones for each rollup which can be used for resource optimization. 

Screenshot 2023-10-13 at 1 40 22 PM

**Review by estherk15 — APPROVED** [review]

**Pull request merged**